onward/app/feed/MostViewedGalleryAgent.scala

```diff
@@ -32,7 +27,7 @@ object MostViewedGalleryAgent extends Logging with ExecutionContexts {
         url <- (item \ "url").asOpt[String]
         count <- (item \ "count").asOpt[Int]
       } yield {
-        getResponse(LiveContentApi.item(UrlToContentPath(url), Edition.defaultEdition)).map(_.content.map(Content(_)))
```

Comment:
The curse of refactoring ... you get made to refactor everything surrounding your refactor. :wink: